bib/cmd/bootc-image-builder/main.go

```diff
@@ -234,6 +243,9 @@ func manifestFromCobra(cmd *cobra.Command, args []string) ([]byte, *mTLSConfig,
 		logrus.Debug("Using local container")
 	}
 
+	pbar.SetPulseMsgf("Manifest generation step")
+	pbar.Start()
```

Comment:
There's no `defer pbar.Stop()` which can lead to "leaking".

Reply:
I tweaked this and added the following comment:
```
manifestFromCobra generate an osbuild manifest from a cobra
commandline and it also takes a progress.ProgressBar as input.

It needs an unstarted progres bar and will start it at the right
point (it cannot be started yet to avoid the "podman pull" progress
and our progress fighting). The caller is responsible for stopping
the progress bar (this functtion cannot know what else needs to happen
after manifest generation).

TODO: provide a podman progress reader to integrate the podman progress
into our progress.
```

I can look into tweaking this further, right now the issue is that `manifestFromCobra` is calling `podman pull` and here we do want to also provide a progress stream which we currently just get directly from podman. So we cannto yet start our own progress (or the two would fight over the screen). 

I guess an alternative would be to split manifestFromCobra into pullContainer and generateManifest or something and then the pullContainer would not take a progress but generateManifest would and that can then be started too (or we would even remove start as any creation of a progress would automatically start it).

I can look into this Monday if you feel the existing comment/structure is still murky .